docs/scos/dev/setup/installing-spryker-with-development-virtual-machine/scos-supported-browsers.md

```diff
@@ -0,0 +1,16 @@
+---
+title: SCOS supported browsers
+description: This document lists browsers which the Spryker Commerce OS supports.
+last_updated: Jun 8, 2022
+template: howto-guide-template
+---
+
+This document lists browsers which the Spryker Commerce OS supports.
+
+### Supported browsers
+
+The Spryker Commerce OS supports the following browsers for all frontend-related projects and products—[B2B Demo Shop](/docs/scos/user/intro-to-spryker/b2b-suite.html), [B2C Demo Shop](/docs/scos/user/intro-to-spryker/b2c-suite.html), [Master Suite](/docs/scos/user/intro-to-spryker/master-suite.html):
+
+| DESKTOP (YVES AND ZED) | MOBILE (YVES ONLY) | TABLET (YVES ONLY) |
+| --- | --- | --- |
+| *Browsers*: <ul><li> Windows, macOS: Chrome (latest version)</li> <li>Windows: Firefox (latest version)</li><li>Windows: IE 11</li><li>Windows: Edge (latest version)</li><li>macOS: Safari (latest version)</li></ul> *Windows versions*:<ul><li>Windows 10</li><li>Windows 7</li></ul>*macOS versions*:<ul><li> Catalina (basically, the latest released version)</li></ul>*Screen resolutions*:<ul><li>1024-1920 width</li></ul>|*Browsers*: <ul><li>iOS: Safari</li><li>Android: Chrome</li></ul>*Screen resolutions*:<ul><li>360x640—for example, Samsung Galaxy S8 or S9</li><li>375x667—for example, iPhone 7 or 8</li><li>iPhone X, Xs, Xr</li></ul>*Android versions*:<ul><li>8.0</li></ul>*iOS versions*:<ul><li>iOS 13 (basically, the latest released version)</li></ul> | *Browsers*: <ul><li>iOS: Safari</li><li>Android: Chrome</li></ul>*iOS versions*:<ul><li>iOS 13</li></ul>*Screen resolutions*:<ul><li>1024x703—for example, iPad Air</li></ul>|
```

Comment:
Here you have Desktop / Mobile / Tablet, but on Marketplace/Merchant Portal page, you have Desktop / Tablet / Mobile.
We need to make the table with the same structure.